### PR TITLE
Convert default JAX-RS 2.1 JSON provider to use the jsonp-1.1 feature

### PR DESCRIPTION
**Pull request by WhiteCat22**
Signed-off-by: Adam Anderson <atanders@us.ibm.com>

### CODE REVIEW

**Comment by WhiteCat22** [comment]
I ran the changes against our existing FAT test suites. There should be no differences from an external perspective.

**Comment by WhiteCat22** [comment]
#build

File: dev/com.ibm.ws.jaxrs.2.1.common/src/com/ibm/ws/jaxrs20/providers/jsonp/JsonPProvider.java [hunk 1]
```diff
@@ -33,177 +36,89 @@
...
-            }
-        }
+    public JsonPProvider(JsonProvider jsonProvider) {
+        this.jsonProvider = jsonProvider;
     }
 
     @Override
-    public long getSize(Object arg0, Class arg1, Type arg2, Annotation[] arg3, MediaType arg4) {
-
+    public long getSize(Object obj, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
```
Comment (andymc12) [inline dev/com.ibm.ws.jaxrs.2.1.common/src/com/ibm/ws/jaxrs20/providers/jsonp/JsonPProvider.java]: The `getSize` method is now a default interface method, so in JAX-RS 2.1, we don't need to implement/override it here.  I'm fine delivering this as-is, but let's remove this method when we make a further change to this class.

**Comment by WhiteCat22** [comment]
#40

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Pull request merged**